### DISCUSSION TITLE
pjit of jit'ed function?

### DISCUSSION THREAD

**Discussion by mattiasmar** [jax-ml/jax#9224] [Q&A]
What is the effect of pjit'ing an already jit'ed method? 
This could be the case when one pjit's an high level method that 'hides' within it a jit'ed method (implemented by another programmer in a library that you can't / don't want to edit).

**Comment by AdrienCorenflos** [comment]
:eyes: (I'm also interested in the converse: jit of pjit'ed function)

**Reply by mattiasmar** [reply]
It is my understanding that jit'ing a pjit'ed function wouldn't have an impact as pjit already causes the code to be compiled.
